File: doc/_i18n/jp/tutorials/introduction/com-controller/create_com_task_sources.py
# mc_tasksモジュールをインポートする
import mc_tasks
# コンストラクター内でタスクを作成して問題に追加する
self.comTask = mc_tasks.CoMTask(self.robots(), 0, 10.0, 1000.0)
self.qpsolver.addTask(self.comTask)
# 姿勢制御タスクの剛性を下げる
self.postureTask.stiffness(1)
# resetコールバック関数内で、現在の質量中心に合わせてタスクをリセットする
self.comTask.reset()

File: doc/_i18n/jp/tutorials/introduction/com-controller/create_com_task_sources.cpp
// 質量中心タスクのヘッダーをインクルードする（ヘッダー）
#include <mc_tasks/CoMTask.h>
// クラス内のプライベートメンバー（ヘッダー）
std::shared_ptr<mc_tasks::CoMTask> comTask;
// コンストラクター内でタスクを作成して問題に追加する
comTask = std::make_shared<mc_tasks::CoMTask>(robots(), 0, 10.0, 1000.0);
solver().addTask(comTask);
// 姿勢制御タスクの剛性を下げる
postureTask->stiffness(1);
// reset関数内で、現在の質量中心に合わせてタスクをリセットする
comTask->reset();
